index.txt

```diff
@@ -39,16 +39,16 @@ install_caddy()
 
 	# Termux on Android has $PREFIX set which already ends with /usr
 	if [[ -n "$ANDROID_ROOT" && -n "$PREFIX" ]]; then
-		install_path="${PREFIX}/bin"
+		install_path="$PREFIX/bin"
```

Comment:
I've been told using the `${BRACE_SYNTAX}` is safer - removing them just for readability?